potodo/potodo.py

```diff
@@ -267,6 +268,8 @@ def main():
         help="Render list with the count of remaining entries (translate or review) rather than percentage done",
     )
 
+    parser.add_argument('-V', '--version', action='version', version='%(prog)s ' + __version__)
```

Comment:
```suggestion
    parser.add_argument('--version', action='version', version='%(prog)s ' + __version__)
```
No need for short option here